.github/workflows/tests.yml

```diff
@@ -0,0 +1,29 @@
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          repository: alphagov/govuk-content-schemas
+          ref: deployed-to-production
+          path: govuk-content-schemas
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: .ruby-version
+      - uses: actions/cache@v1
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+
+      - run: |
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
```

Comment:
It seems a bit of a shame for us to propagate these same magic numbers around every build. I can see they come from the action examples, wonder if they could be persuaded to set up env vars to specify the default. 

By the way, not relevant to this repo but for repos with gems in groups we should use the --with=test to only install test gems.